tests/cli/test_cmds_status.py

```diff
@@ -864,7 +866,7 @@ def test_status_with_registration(
         service_instance=None,
         new=False,
     )
-    return_value = paasta_status(args)
+    return_value, lock = paasta_status(args)
```

Comment:
There are a few ways you could write this test (and the other test that follows the same pattern) without having `paasta_status` return the lock object:

# mocking

You could change these lines to:
```
with mock.patch("paasta_tools.cli.cmds.status.Lock", autospec=True) as mock_lock:
    return_value = paasta_status(args)
```
which would make it so when `paasta_status` calls `Lock()`, it's calling a MagicMock object instead (which would return another MagicMock.)

then in your `assert_called_once_with` call, you'd specify `lock=mock_lock()`.

You could also pre-define a fake lock object:

```
fake_lock = mock.MagicMock()
with mock.patch("paasta_tools.cli.cmds.status.Lock", autospec=True, return_value=fake_lock):
    return_value = paasta_status(args)
```
then later, you'd pass `lock=fake_lock` to `assert_called_once_with`.

# mock.ANY

You could just specify `lock=mock.ANY` on line 883, which would make assert_called_once_with not care about the specific value of the `lock` argument.